src/Webapi/Webapi__IntersectionObserver.res

```diff
@@ -0,0 +1,44 @@
+/**
+ * Spec: https://www.w3.org/TR/intersection-observer/
+ */
+module IntersectionObserverEntry = Webapi__IntersectionObserver__IntersectionObserverEntry
```

Comment:
Does it need to be a separate file, would the entry module be used elsewhere? Could it just be a local module?

Reply:
Was just following the pattern from the `ResizeObserver`, I don't believe this would be used anywhere else.

Reply:
I don't really have strong feelings either way here -- would you like it to be local?

Reply:
Yes, I think so. Every file is visible from outside the project, so I'd like to reduce them where it makes sense to. I'll probably swap `ResizeObserver` to a local module at some point.